client/lib/plans/constants.js

```diff
@@ -181,6 +181,14 @@ export const plansList = {
 			FEATURE_GOOGLE_ANALYTICS,
 			FEATURE_NO_BRANDING
 		] ),
+		getPromotedFeatures: () => [
```

Comment:
Doesn't have to be all of them, but perhaps some of the latest things we think are big incentives.